docs/main/03-reference/16-health-checks/_assets/basic-definition

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
+
+set -eux
+
+cat /kratix/input/object.yaml
+
+resourceName=$(yq '.metadata.name' /kratix/input/object.yaml)
+namespace="$(yq '.metadata.namespace // "default"' /kratix/input/object.yaml)"
+promiseName="redis"
+
+cat <<EOF > /kratix/output/health-definition.yaml
+apiVersion: platform.kratix.io/v1alpha1
+kind: HealthDefinition
+metadata:
+  name: ${promiseName}-${resourceName}-${namespace}
+  namespace: default
+spec:
+  resourceRef:
+    name: ${resourceName}
+    namespace: ${namespace}
+  promiseRef:
+    name: ${promiseName}
+  # runs every minute
+  schedule: "* * * * *"
+  input: |
+    name: ${resourceName}
+  workflow:
+    apiVersion: platform.kratix.io/v1alpha1
+    kind: Pipeline
+    metadata:
+      name: healthcheck
+    spec:
+      containers:
+      - image: busybox
+        name: check-redis
```

Comment:
Is it possible to use a container name that's more aligned with health check than busybox?

Reply:
added a comment to clarify